tests/integration_rust/add_tests.rs

```diff
@@ -96,6 +96,10 @@ async fn add_with_channel() {
         .await
         .unwrap();
 
+    pixi.project_channel_add()
+        .with_channel("https://prefix.dev/conda-forge")
+        .await
```

Comment:
Maybe keep the frozen here, but remove it in the second one? As the it will then use a solve for both channels?